ui/src/app/machines/views/MachineDetails/MachineNetwork/NetworkTable/NetworkTable.tsx

```diff
@@ -58,184 +66,240 @@ type SortKey = keyof NetworkRowSortData;
 const getSortValue = (sortKey: SortKey, row: NetworkRow) =>
   row.sortData[sortKey];
 
-const generateRows = (
+const generateRow = (
+  nic: NetworkLinkInterface,
   machine: Machine,
   fabrics: Fabric[],
   vlans: VLAN[],
   fabricsLoaded: boolean,
   vlansLoaded: boolean
-): NetworkRow[] => {
```

Comment:
Despite what this diff shows, all these changes in the `generateRow` function are indentation changes (content moved from generateRow**s** to generateRow).